### PR TITLE
Release Godot 4.1 RC 3 and publish release notes

### PR DESCRIPTION
**Pull request by akien-mga**

### CODE REVIEW

File: collections/_article/release-candidate-godot-4-1-rc-3.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.1.rc3/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+-----
+
+*The illustration picture for this article is from* **Bravest Coconut**](https://store.steampowered.com/app/2320040/Skies_above_the_Great_War/), *a "classic adventure game featuring a world full of zany characters to meet, spooky monsters to fight, and brain teasing puzzles to solve". The game is developed by Nathan Hoad ([Twitter](https://twitter.com/nathanhoad), [Mastodon](https://mastodon.social/@nathanhoad)) and Lilly Piri ([Twitter](https://twitter.com/lillypiri), [Mastodon](https://mastodon.social/@lillypiri)) using Godot 4. Nathan also makes Godot plugins such as [Dialogue Manager](https://github.com/nathanhoad/godot_dialogue_manager), and publishes devlogs and Godot tutorials on [YouTube](https://www.youtube.com/@nathan_hoad).*
```
Comment (YuriSizov) [inline collections/_article/release-candidate-godot-4-1-rc-3.md]: Why is it in quotation marks? Also the link to the game is wrong :)

**Review by YuriSizov — APPROVED** [review]
Aside from the illustration blurb it's all good. 👍

File: collections/_article/release-candidate-godot-4-1-rc-3.md (hunk 1 above)
Comment (akien-mga) [reply collections/_article/release-candidate-godot-4-1-rc-3.md]: I copied what Nathan told me in a DM verbatim, and it sounded like the game's elevator pitch. Felt weird to just copy a marketing pitch as is. I'll rephrase it.

Comment (akien-mga) [reply collections/_article/release-candidate-godot-4-1-rc-3.md]: Fixed.

**Pull request merged**